docker/Dockerfile.build.aarch64-qnx

```diff
@@ -5,6 +5,7 @@ FROM ${QNX_BASE_IMAGE}
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
A side note. I think we can remove this file at all as we don't support QNX anymore.